find_primes.py

```diff
@@ -1 +1,29 @@
+#num1 is the original number, num2 is the prime number, num3 is all of the primes,
+num1 = int(input("Enter any positive integer: "))
+num2 = num1 + 1
+num3 = []
+count = 0
+
+def find_prime(a):
+  '''find_prime(a) -> yes or no
+  Prints whether a is a prime'''
+  import math
+  limit = int((math.sqrt(a)) + 1)
+  for i in range(2, limit):
+    value1 = a / i
+    value2 = int(a / i)
+    if value1 == value2:
+      return "no"
+  return "yes"
+
+while count != 5:
+  result = find_prime(num2)
+  if result == "yes":
```

Comment:
If you change the function to return true instead of “yes”, then this line can just be:

`if result:`